WordPress/Classes/ViewRelated/System/Coordinators/MySitesCoordinator.swift

```diff
@@ -56,14 +44,9 @@ class MySitesCoordinator: NSObject {
         splitViewController.restorationIdentifier = MySitesCoordinator.splitViewControllerRestorationID
         splitViewController.presentsWithGesture = false
         splitViewController.setInitialPrimaryViewController(navigationController)
-        splitViewController.dimsDetailViewControllerAutomatically = !FeatureFlag.newNavBarAppearance.enabled
+        splitViewController.dimsDetailViewControllerAutomatically = false
```

Comment:
isn't this property `false` by default? If so, can we remove this line?

Reply:
I'll check and remove if it is.

Reply:
Done